src/lcm/entry_point.py

```diff
@@ -28,7 +30,7 @@ def get_lcm_function(
     *,
```

Comment:
Can't we use the enum here? Also, I'd prefer getting rid of the default value.

Reply:
We can get rid of the default value, but we cannot use the `Enum` here for two reasons:

1. The `Enum` does not work for "solve_and_simulate". This is something we should discuss anyways: Should there even be a "solve_and_simulate" option? Or should there only be "solve" and "simulate", and if someone wants a simulation function that does not automatically do the solving step we add a boolean argument?
2. I would not want the user to import that `Enum`, but rather try to convert the user input into an `Enum` during the input processing.

Reply:
Cool, makes sense.

I do think there should be a one-stop-shop, but via a separate function that calls `solve` and `simulate` in order. See my comment below in the interface to `simulate`.